src/types/Path.ts

```diff
@@ -0,0 +1,7 @@
+export enum PATH {
```

Comment:
Not a good practice to use an enum for values. Use a constant for this purpose. Or use enum values as keys for constant.
```ts
enum Path {
  Main = 'Main';
}

type AppPaths = Record<Path, string>;

const PATHS: AppPaths = {
  [Path.Main]: '/',
}
```

Don't forget to separate constants and types.